src/main/java/it/gov/pagopa/pu/bff/dto/OffsetDateTimeIntervalFilter.java

```diff
@@ -0,0 +1,20 @@
+package it.gov.pagopa.pu.bff.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.io.Serializable;
+import java.time.OffsetDateTime;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class OffsetDateTimeIntervalFilter implements Serializable {
+
+  private OffsetDateTime paymentDateTimeFrom;
+  private OffsetDateTime paymentDateTimeTo;
```

Comment:
This is a generic DTO
```suggestion
  private OffsetDateTime from;
  private OffsetDateTime to;
```